app/manage/pods/common/sms-factory.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+module.exports = [
+  '$http',
+  function($http) {
+    var urlBase = 'http://concaria-sms.herokuapp.com/api';
```

Comment:
We can change how the proxy works later, but it would require more effort, since the underlying idea is just wrong.